Packs/AbnormalSecurity/Integrations/AbnormalSecurity/AbnormalSecurity.py

```diff
@@ -345,7 +367,7 @@ def get_employee_identity_analysis_genome_data_command(client, args):
     headers = ['description', 'key', 'name', 'values']
 
     markdown = tableToMarkdown(
-        f"Analysis of {email_address}", response.get('histograms', []), headers=headers, removeNull=True)
+        f"Analysis of {email_address}", response.get('data', []), headers=headers, removeNull=True)
```

Comment:
Why did you change it? Has the API changed?

Reply:
@MosheEichler the API did not change. the response always returned with field name `data` and we are fixing this bug with the xsoar logic.